public/sass/components/_dashboard_grid.scss

```diff
@@ -85,6 +85,15 @@
   }
 }
 
+// Hack to prevent panel overlap during drag/hover (due to descending z-index assignment)
+.react-grid-item {
+  &:hover,
+  &:active,
+  &:focus {
+    z-index: 100 !important;
+  }
+}
+
```

Comment:
if i remember correctly the dashboards squad had a preference for us to avoid modifying .scss and adding these changes in useStyles instead. As this has been blocked for a while I'd be ok moving forward with this implementation and making a note to clean these styles up once the project is unblocked 👍

Reply:
im not sure it makes sense to keep some grid-related styles in scss and others in useStyles. i imagine one strategy in a single place is preferable to two in different places.

if the plan is to move away from scss to useStyles, then it should probably be holistic and not piecemeal.

Reply:
Think it's good here, contained with orther grid styles.